### DISCUSSION TITLE
Ooops 500 internal server error when clicking new pullrequest

### DISCUSSION THREAD

**Discussion by creamy11** [community/community#48183] [Pull Requests]
### Select Topic Area

Bug

### Body

I keep getting "Ooops 500" error when trying to create a new pull request on `creamy11/cost_effective_hzvac` and `creamy11/metapop_kor`.

I thought it was because I uploaded big files, so I upgraded my storage plan, but I'm still getting the same error.

<img width="1680" alt="Screen Shot 2023-02-21 at 12 30 11 PM" src="https://user-images.githubusercontent.com/22143967/220421484-d08233ba-dd03-4070-8d4a-50f8d257d4c7.png">

**Comment by 0-v-0** [comment]
I have this problem too when trying to create a new pull request [from 0-v-0/bcaa to aferust/bcaa](https://github.com/aferust/bcaa/compare/master...0-v-0:bcaa:patch?expand=1). Did you manage to solve it?
